florist/api/monitoring/metrics.py

```diff
@@ -0,0 +1,57 @@
+"""Classes for the instrumentation of metrics reporting from clients and servers."""
+import json
+from logging import DEBUG
+from typing import Any, Dict, Optional
+
+import redis
+from fl4health.reporting.metrics import DateTimeEncoder, MetricsReporter
+from flwr.common.logger import log
+
+
+class RedisMetricsReporter(MetricsReporter):  # type: ignore
```

Comment:
curious why we have to type ignore this

Reply:
ahhh no output folder

Reply:
It was this weird error:
https://stackoverflow.com/questions/49888155/class-cannot-subclass-qobject-has-type-any-using-mypy

I tried solving it a couple of different ways without luck, so I ignored it. Seems to be something mypy is not picking up from fl4health's package settings and not really a big issue. Not sure if it's worth fixing it in fl4health. We can also disable this check if it becomes too annoying.